python/src/main/python/drivers/vkrun.py

```diff
@@ -15,67 +15,78 @@
 # limitations under the License.
 
 import argparse
+import filecmp
 import os
+import shutil
 import subprocess
 import time
-import filecmp
-import shutil
+from typing import Union, IO, Any
 
 ################################################################################
 # Constants
 
+
 LOGFILE = 'vklog.txt'
 TIMEOUT_RUN = 30
 NUM_RENDER = 3
```

Comment:
I like the above and it would be good to do the same for other constants (e.g. timeout)